### PR TITLE
WF-45111-Add UG Documentation to DisplayBackgrounds property and remove existing background

### PR DESCRIPTION
**Pull request by elizabethatienoo**
Added UG documentation to md fileformat to hide and remove backgrounds in Word document.

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 1]
```diff
@@ -1432,6 +1432,210 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+        //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:1535]: Add github link once merged

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 2]
```diff
@@ -1432,6 +1432,210 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+## Remove background in Word document
+
+You can remove background colors and images in Word document using `NoBackground` API as background type.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:1539]: You can remove background colors and images in an existing Word document by setting `NoBackground` as background type.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 3]
```diff
@@ -1432,6 +1432,210 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+        //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:1636]: Add link once merged

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 2 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: This have been corrected.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 4]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Apply-picture-background-to-document).
 
+## Hide background in print layout view
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1435]: ## Hide background in the print layout view

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 5]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+You can show or hide background colors and images in print layout view of Word document using `DisplayBackgrounds` API.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1437]: You can show or hide background colors and images in the print layout view of a Word document using the `DisplayBackgrounds` API.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 6]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+The following code example shows how to hide background in print layout view of Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1439]: The following code example shows how to hide the background in the print layout view of a Word document:

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 7]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load Word document.
+using (WordDocument document = new WordDocument(“Input.docx” FormatType.Docx))
+{
+//Disable a flag to hide background in print layout view.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1447]: //Disable a flag to hide the background in the print layout view.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 8]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+//Save the Word document.
+document.Save(“Sample.docx”), FormatType.Docx);
+}
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load Word document.
+Using document As WordDocument = New       WordDocument(“Input.docx"), FormatType.Docx)
+'Disable a flag to hide background in print layout view.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1458]: 'Disable a flag to hide the background in the print layout view.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 9]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+'Disable a flag to hide background in print layout view. 
+document.Settings.DisplayBackgrounds = False
+'Save the Word document.
+document.Save(“Sample.docx"), FormatType.Docx)
+End Using
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//"App" is the class of Portable project.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1467]: //"App" is the class of Portable projects.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 10]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Disable a flag to hide background in print layout view.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1475]: //Disable a flag to hide the background in the print layout view.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 11]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Settings.DisplayBackgrounds = false;
+        //Save the Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as Word document file in local machine.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1480]: //Save the stream as a Word document file on the local machine.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 12]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        Save(stream, "Sample.docx");
+        //Please refer the below link to save Word document in UWP platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1482]: //Please refer to the below link to save the Word document in the UWP platform

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 13]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream docStream = new FileStream("Input.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Disable a flag to hide background in print layout view.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1496]: //Disable a flag to hide the background in the print layout view.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 14]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Settings.DisplayBackgrounds = false;
+        //Save the Word document to MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download Word document in the browser.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1502]: //Download the Word document in the browser.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 15]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1515]: //Load the file stream into the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 16]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        outputStream.Position = 0;
+        //Download Word document in the browser.
+        return File(outputStream, "application/msword", "Sample.docx"); 
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1510]: //"App" is the class of Portable projects.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 17]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Disable a flag to hide background in print layout view.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1518]: //Disable a flag to hide the background in the print layout view.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 18]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Settings.DisplayBackgrounds = false;
+        //Save the Word document to MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing.
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+        //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1525]: //Please download the helper files from the  link below to save the stream as a file and open the file for viewing on Xamarin platform

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 19]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
+
+## Remove background in Word document
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1536]: ## Remove background in a Word document

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 20]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+You can remove background colors and images in an existing Word document by setting `NoBackground` as background type.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1538]: You can remove background colors and images in an existing Word document by setting `NoBackground` as the background type.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 21]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+The following code example shows how to remove background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1540]: The following code example shows how to remove the background in a Word document:

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 22]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load Word document.
+using (WordDocument document = new WordDocument(“Input.docx” FormatType.Docx))
+{
+//Remove the existing background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1548]: //Remove the existing background in the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 23]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+//Save the Word document.
+document.Save(“Sample.docx”), FormatType.Docx);
+}
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load Word document.
+Using document As WordDocument = New       WordDocument(“Input.docx"), FormatType.Docx)
+'Remove the existing background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1559]: 'Remove the existing background in the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 24]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+document.Background.Type = BackgroundType.NoBackground;
+'Save the Word document.
+document.Save(“Sample.docx"), FormatType.Docx)
+End Using
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//"App" is the class of Portable project.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1568]: //"App" is the class of Portable projects.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 25]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Remove the existing background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1576]: //Remove the existing background in the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 26]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Background.Type = BackgroundType.NoBackground;
+        //Save the Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as Word document file in local machine.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1581]: //Save the stream as a Word document file on the local machine.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 27]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        Save(stream, "Sample.docx");
+        //Please refer the below link to save Word document in UWP platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1583]: //Please refer to the link below to save a Word document in the UWP platform

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 28]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+    }
+} 
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream docStream = new FileStream("Input.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load file stream into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1594]: //Load the file stream into the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 29]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Remove the existing background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1597]: //Remove the existing background in the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 30]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Background.Type = BackgroundType.NoBackground;
+        //Save the Word document to MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download Word document in the browser.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1603]: //Download the Word document in the browser.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 31]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        return File(outputStream, "application/msword", "Sample.docx"); 
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1611]: //"App" is the class of Portable projects.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 32]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load file stream into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1616]: //Load the file stream into the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 33]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Remove the existing background in Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1619]: //Remove the existing background in the Word document.

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 34]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+        document.Background.Type = BackgroundType.NoBackground;
+        //Save the Word document to MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing.
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+        //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1626]: //Please download the helper files from the link below to save the stream as a file and open the file for viewing on the Xamarin platform

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 35]
```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
...
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
+
+
 ## Working with Alternate chunks
 
 Updating Alternate chunk in the Word document, imports the content from the embedded alternate chunk into the main document. When saving the Word document containing alternate chunk as DOCX format document, the alternate chunk content preserved by default. But, when saving as DOC format or other formats, the alternate chunk content will not be preserved. You can use [UpdateAlternateChunks](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WordDocument.html#Syncfusion_DocIO_DLS_WordDocument_UpdateAlternateChunks) method to preserve the alternate chunk content by importing into the main document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Word-document.md:1640]: Updating the Alternate chunk in the Word document imports the content from the embedded alternate chunk into the main document. When saving the Word document containing an alternate chunk as a DOCX format document, the alternate chunk content is preserved by default. But, when saving as DOC format or other formats, the alternate chunk content will not be preserved. You can use [UpdateAlternateChunks](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WordDocument.html#Syncfusion_DocIO_DLS_WordDocument_UpdateAlternateChunks) method to preserve the alternate chunk content by importing into the main document.

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 4 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 5 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 6 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Addressed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 7 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Addressed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 8 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 9 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 10 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 11 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 12 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 13 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 14 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 35 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md:1641]: This is an existing change

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 34 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 33 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 32 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 31 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 30 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 29 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 28 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 27 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 26 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 25 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 24 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 23 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 22 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 20 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 19 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 18 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 17 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 16 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 15 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md]: Done

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**